packages/api/src/brevo-contact-log/entity/brevo-contact-log.entity.factory.ts

```diff
@@ -0,0 +1,56 @@
+import { DocumentInterface } from "@comet/cms-api";
+import { Embedded, Entity, OptionalProps, PrimaryKey, Property } from "@mikro-orm/core";
+import { Type } from "@nestjs/common";
+import { Field, ID, ObjectType } from "@nestjs/graphql";
+import { v4 } from "uuid";
+
+import { EmailCampaignScopeInterface } from "../../types";
+
+export interface BrevoContactLogInterface {
+    email: string;
+    userId: string;
+    scope: EmailCampaignScopeInterface;
+    createdAt: Date;
+    updatedAt: Date;
+}
+
+export function createBrevoContactLogEntity({ Scope }: { Scope: EmailCampaignScopeInterface }): Type<BrevoContactLogInterface> {
+    @Entity()
+    @ObjectType({
+        implements: () => [DocumentInterface],
+    })
+    class BrevoContactLog implements BrevoContactLogInterface, DocumentInterface {
+        [OptionalProps]?: "createdAt" | "updatedAt";
+
+        @PrimaryKey({ columnType: "uuid" })
+        @Field(() => ID)
+        id: string = v4();
+
+        @Property({ columnType: "text" })
+        @Field()
+        email: string;
+
+        @Property()
+        @Field()
+        userId: string;
```

Comment:
Am i correct here:
 - email is from the user that was created in brevo
 - userId is the user that created the new brevo contact

We should clarify that more. Suggestions:
 - importedEmail/...
 - responsibleUserId/adminUserId/...